services/hanke-service/src/main/resources/application.properties

```diff
@@ -1,15 +1,14 @@
 haitaton.cors.allowedOrigins=${HAITATON_CORS_ALLOWED_ORIGINS:http://localhost:3001}
-
+# Allu properties
```

Comment:
If we want to divide the properties into strict sections, maybe a YAML file would be better?